operator/controller/src/test/java/io/apicurio/registry/operator/it/ITBase.java

```diff
@@ -157,6 +163,15 @@ private static void registerReconcilers() {
         }
     }
 
+    public static void configureLocalOperator() {
+        if (ConfigProvider.getConfig().getOptionalValue("registry.app.image", String.class).isEmpty()) {
```

Comment:
let see if we can consolidate fetching those properties, at least, from static methods

Reply:
Done, replaced by defaults in the `application.properties`.